PendingReleaseNotes

```diff
@@ -283,6 +283,14 @@ CephFS: Disallow delegating preallocated inode ranges to clients. Config
   which specifies the path within the CephFS to mount this export on. If this and the other
   `EXPORT { FSAL {} }` options are the same between multiple exports, those exports will share a single CephFS client. If not specified, the default is `/`.
 
+* CephFS: `fs subvolume create` command gained `--earmark` option. Support for
+  adding earmark to subvolume is added to CephFS in Squid 19.2.1.
+  The earmark is a unique identifier that tags the subvolume for specific purposes,
+  such as NFS or SMB services. By default, no earmark is set, allowing for flexible
```

Comment:
> NFS or SMB services.

Are there more services that user can earmark a subvol for? If there is such a list maintained can you share (for the benefit of UI)? If its just these two, then rather than being ambiguous in the release notes should we just be specific about the services that can be earmarked?

Reply:
yup